images/access-setup/Dockerfile

```diff
@@ -4,15 +4,16 @@ RUN mkdir /workspace && chmod 777 /workspace && chown 65532:65532 /workspace
 ENV HOME /tmp/home
 RUN mkdir $HOME && chmod 777 $HOME && chown 65532:65532 $HOME
 
-RUN microdnf install gzip tar
+RUN microdnf install -y gzip-1.9 tar-1.30 && microdnf clean all
```

Comment:
No action: I understand the reasoning: reproducible builds. The approach is however only workable with image scanning that we will have soon in place. We would otherwise get staled binaries with an increasing amount of CVEs.

Reply:
As I am mentioning image scanning we should use package manager whenever possible like for jq, kubectl and ShellCheck below. Otherwise the scanner may not be able to identify CVEs in these binaries. I will open an issue for that